Bottom: b7ed844e934a2a1e5c9e53f78ff7264c835a5a0d
Top:    b3a8c67d2f2c90112f117382aa7feefa68c61efe
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2010-12-28 01:47:20 +0000

Refresh of drop-in-pcb-gl-code-various-mess

---

diff --git a/src/hid/gtk/gtkhid-gl.c b/src/hid/gtk/gtkhid-gl.c
index e75d36b..332deb5 100644
--- a/src/hid/gtk/gtkhid-gl.c
+++ b/src/hid/gtk/gtkhid-gl.c
@@ -521,7 +521,7 @@ ghid_set_line_cap_angle (hidGC gc, int x1, int y1, int x2, int y2)
   printf ("ghid_set_line_cap_angle() -- not implemented\n");
 }
 
-void
+static void
 ghid_invalidate_current_gc (void)
 {
   current_gc = NULL;
diff --git a/src/hid/gtk/gui.h b/src/hid/gtk/gui.h
index 77e4369..d8befcf 100644
--- a/src/hid/gtk/gui.h
+++ b/src/hid/gtk/gui.h
@@ -485,7 +485,6 @@ void ghid_set_line_width (hidGC gc, int width);
 void ghid_set_draw_xor (hidGC gc, int xor);
 void ghid_set_draw_faded (hidGC gc, int faded);
 void ghid_set_line_cap_angle (hidGC gc, int x1, int y1, int x2, int y2);
-void ghid_invalidate_current_gc (void);
 void ghid_draw_line (hidGC gc, int x1, int y1, int x2, int y2);
 void ghid_draw_arc (hidGC gc, int cx, int cy, int xradius, int yradius,
                     int start_angle, int delta_angle);
